myst_nb/converter.py

```diff
@@ -248,6 +258,19 @@ def _flush_markdown(start_line, token, md_metadata):
         if token.type == "fence" and token.info.startswith(code_directive):
             _flush_markdown(md_start_line, token, md_metadata)
             options, body_lines = read_fenced_cell(token, len(notebook.cells), "Code")
+            # Parse :file or file: tags and populate body with contents of file
+            if "file" in options:
```

Comment:
```suggestion
            if "myst_file" in options:
```

Reply:
@chrisjsewell can you please explain why you'd like `myst_file` (as the files usually have no relationship with `myst` files)?

Reply:
see above